.github/workflows/build.yml

```
@@ -60,7 +60,7 @@ jobs:
     - name: archive
       run: cd .. && tar --exclude-vcs -zcf ${{ matrix.GOOS }}-${{ matrix.GOARCH }}.tar.gz go
     - name: save
-      uses: actions/upload-artifact@v1
+      uses: actions/upload-artifact@c24449f33cd45d4826c6702db7e49f7cdb9b551d # v4.4.0 
```

Whoops had the wrong sha in my clipboard:
```suggestion
      uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0 
```